community/kernel/src/main/java/org/neo4j/kernel/internal/KernelDiagnostics.java

```diff
@@ -93,15 +95,25 @@ private long logStoreFiles( Logger logger, String prefix, File dir )
 
             // Sort by name
             List<File> fileList = Arrays.asList( files );
-            Collections.sort( fileList, new Comparator<File>()
+            if ( fileList.size() > MAX_DIRECTORY_LISTING_THRESHOLD )
```

Comment:
i think we need to mention in the logs what the order is to avoid our own confusion during investigation:
Like:
```Bla ...bla.. store files in accordance to their store size``` or similar